iree/compiler/Dialect/HAL/Conversion/HALToVM/ConvertHALToVM.cpp

```diff
@@ -138,7 +138,7 @@ std::unique_ptr<OperationPass<ModuleOp>> createConvertHALToVMPass(
 }
 
 static PassRegistration<ConvertHALToVMPass> pass([] {
-  auto options = IREE::VM::getTargetOptionsFromFlags();
+  auto options = IREE::VM::TargetOptions::FromFlags::getRegistered();
```

Comment:
`getRegistered()` reads really weird to me as I'd think that meant "get the list of registered options" vs "get the options that are registered" - maybe just rename to `get()`?

Reply:
Thanks - I'll make that change.